src/Microsoft.Data.SqlClient/ref/System.Data.Common.Batch.IAsyncDisposable.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace System.Data.Common
+{
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+    public abstract partial class DbBatch : System.IAsyncDisposable
```

Comment:
Since we are only implementing the new batch APIs in netcore, we don't need this file at all, right?

Reply:
removed.